### PR TITLE
add tail_latest_file

### PR DESCRIPTION
**Pull request by tkrol88**

### CODE REVIEW

File: moler/cmd/unix/tail_latest_file.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                      'last_file=""\n' \
+                      'tail_pid=""\n' \
+                      'while :\n' \
+                      'do\n' \
+                      'current_last_file=`ls -t {} | head -1`\n' \
+                      'if [ "$last_file" != "$current_last_file" ]\n' \
+                      'then\n' \
+                      '[ -n "$tail_pid" ] && kill $tail_pid\n' \
+                      'last_file=$current_last_file\n' \
+                      'tail -f $last_file &\n' \
```
Comment (marcin-usielski) [inline moler/cmd/unix/tail_latest_file.py:64]: `tail -f $current_last_file` takes only 10 last lines (if file has more lines then they will not be shown) `tail -f -n +1 $current_last_file` should show all lines

File: moler/cmd/commandtextualgeneric.py [hunk 2]
```diff
@@ -140,6 +141,9 @@ def _build_command_string_escaped(self):
         if self.__command_string is not None:
             if self._max_index_from_beginning != 0 or self._max_index_from_end != 0:
                 sub_command_string = self._build_command_string_slice()
+            elif self._multiline_cmd:
+                cmd_without_new_line = re.sub('\n', '', self.__command_string)
+                sub_command_string = re.escape(cmd_without_new_line)
```
Comment (marcin-usielski) [inline moler/cmd/commandtextualgeneric.py:146]: What do you think to use slice like single line command? I mean on some terminals the command echo looks really weird.

Comment (tkrol88) [reply moler/cmd/commandtextualgeneric.py]: Done

File: moler/cmd/unix/tail_latest_file.py (hunk 1 above)
Comment (tkrol88) [reply moler/cmd/unix/tail_latest_file.py]: Done

**Pull request merged**